modules/core/04-channel/keeper/handshake.go

```diff
@@ -82,7 +100,12 @@ func (k Keeper) ChanOpenInit(
 		),
 	})
 
-	return channelID, capKey, nil
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
someone can do a followup pr to add a function `EmitOpenInitEvents` to the `04-channel/keeper/events.go` file for each handshake step